Sources/Validation/Validators/CountValidator.swift

```diff
@@ -75,14 +68,22 @@ fileprivate struct CountValidator<T>: ValidatorType where T: Collection {
     func validate(_ data: T) throws {
         if let min = self.min {
             guard data.count >= min else {
-                throw BasicValidationError("is not larger than \(min)")
+                throw BasicValidationError("is shorter than \(elementDescription(count: min))")
```

Comment:
This statement will not be correct if the the count is equal. I think that's why I used `not larger` initially. 

```suggestion
                throw BasicValidationError("is not longer than \(elementDescription(count: min))")
```

Reply:
Actually, I believe the current message `"is not larger"` is incorrect as `min` is a valid value.

In `validate(:)`, we guard that `data.count >= min`, and `ValidationTests.swift` contains the following:

```swift
try Validator<Int>.range(-5...5).validate(5)
try Validator<Int>.range(-5...5).validate(-5)
XCTAssertThrowsError(try Validator<Int>.range(-5...5).validate(6))
XCTAssertThrowsError(try Validator<Int>.range(-5...5).validate(-6))
```

Reply:
Since we are dealing with numeric values only in `CountValidator`, I personally think that "less than" and "greater than" are the clearest comparator forms.

For extra client error clarity you might also consider more verbose forms along the lines of `is less than required minimum of \(elementDescription(count: min))` and `is greater than required maximum of \(elementDescription(count: max))`.